### PR TITLE
remove hot template reloading

### PR DESCRIPTION
**Pull request by syphar**
In relation to the new web framework I already talked to @jyn514 about this. 
IMHO hot reloading is not needed any more for the kind of changes we do most of the time. 

And I would like to ignore it when working on the new web refactor or anything else.

### CODE REVIEW

**Comment by syphar** [comment]
( also this saves me from trying to upgrade these dependencies) 

r? @Nemo157 or @jyn514 ( only if there is time)

**Review by jyn514 — APPROVED** [review]
Yeah, this looks like it belongs in `cargo watch`. I remember this was originally added before we had `rerun-if-changed` in the build script, so it was constantly rebuilding even if there were no changes to the .rs file - `watch` should be a lot faster now.

File: src/web/page/templates.rs [hunk 1]
```diff
@@ -23,55 +14,21 @@ const TEMPLATES_DIRECTORY: &str = "templates";
 #[derive(Debug)]
 pub(crate) struct TemplateData {
     /// The actual templates, stored in an `ArcSwap` so that they're hot-swappable
```
Comment (Nemo157) [inline src/web/page/templates.rs:16]: This comment is out of date

**Review by Nemo157 — APPROVED** [review]
Other than the one out of date comment, I agree that hot reloading isn't that useful.

File: src/web/page/templates.rs (hunk 1 above)
Comment (syphar) [reply src/web/page/templates.rs]: I removed the comment, good catch!

**Pull request merged**